fontique/src/backend/fontconfig/cache.rs

```diff
@@ -1,65 +1,62 @@
 // Copyright 2024 the Parley Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
-use super::{Stretch, Style, Weight};
 use fontconfig_cache_parser::{Cache, CharSetLeaf, Object, Pattern, Value};
 use std::io::Read;
 use std::path::PathBuf;
+use styled_text::{Stretch, Style, Weight};
 
-impl Stretch {
-    fn from_fc(width: i32) -> Self {
-        match width {
-            63 => Self::EXTRA_CONDENSED,
-            87 => Self::SEMI_CONDENSED,
-            113 => Self::SEMI_EXPANDED,
-            _ => Self::from_ratio(width as f32 / 100.0),
-        }
+// FIXME(style): There's an argument for putting this into styled_text
```

Comment:
I like the idea of `styled_text` having this code as it makes it slightly friendlier from an interop perspective if others start to use it and it centralizes some of the understanding of the values.